### DISCUSSION TITLE
Printer plugin and print media css

### DISCUSSION THREAD

**Discussion by kyllerss** [capawesome-team/capacitor-plugins#350] [Q&A]
I am considering becoming an Insider. One of the plugins that I'd be looking to use is the Printer plugin to print a Webview page. The documentation states that on "Web" I can rely on print media styles to control what gets printed when calling the print webview function. 

Does this not apply to webview-rendered pages on Android or iOS? Does the fact that the documentation specifically calls out "Web" mean that in print media styles is restricted to web pages?

**Comment by robingenz** [comment]
Hi @kyllerss, great questions! Yes, print media styles are also supported on Android and iOS.
